internal/testrunner/common.go

```diff
@@ -0,0 +1,40 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package testrunner
+
+import "os"
+
+// StackSettings encapsulate the various settings required to connect to the Elastic Stack.
+type StackSettings struct {
+	Elasticsearch struct {
+		Host     string
+		Username string
+		Password string
+	}
+	Kibana struct {
+		Host string
+	}
+}
+
+// GetStackSettingsFromEnv loads Elastic stack connnection settings from
+// environment variables and returns them.
+func GetStackSettingsFromEnv() StackSettings {
```

Comment:
I think this is a good moment to put it in the `stack` package and expose it as `stack.CurrentSettings()`.

Reply:
Done in 23fb729.